orange/utils/balancer.lua

```diff
@@ -0,0 +1,293 @@
+---
+-- from https://github.com/Mashape/kong/blob/master/kong/core/balancer.lua
+-- modified by zhjwpku@github
+
+local orange_db = require "orange.store.orange_db"
+local pl_tablex = require "pl.tablex"
```

Comment:
"pl.tablex" what's this? please give a way to solve this deps

Reply:
https://github.com/stevedonovan/Penlight#installation

Reply:
we'd better not import such dependencies like `penlight` for **now**